runtime/dart_isolate.cc

```diff
@@ -657,7 +647,7 @@ DartIsolate::CreateDartVMAndEmbedderObjectPair(
       (*embedder_isolate)->GetIsolateSnapshot()->GetInstructionsIfPresent(),
       (*embedder_isolate)->GetSharedSnapshot()->GetDataIfPresent(),
       (*embedder_isolate)->GetSharedSnapshot()->GetInstructionsIfPresent(),
-      flags, embedder_isolate.get(), error);
+      NULL, embedder_isolate.get(), error);
```

Comment:
Maybe we should keep 'flags' here as they are passed to CreateDartVMAndEmbedderObjectPair as argument.

Reply:
Done.